### PR TITLE
fix create type for data returned from subscription

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +19,8 @@
     "@babel/register": "^7.12.1",
     "@babel/runtime": "^7.12.5",
     "@open-web3/dev-config": "^0.1.13",
-    "@polkadot/api": "^4.16.2",
-    "@polkadot/typegen": "^4.16.2",
+    "@polkadot/api": "^4.14.1",
```
Comment (xlc) [inline package.json:22]: why downgrade?

File: packages/api-mobx/package.json [hunk 2]
```diff
@@ -20,5 +20,8 @@
   },
   "peerDependencies": {
     "@polkadot/api": "4.*"
+  },
+  "devDependencies": {
+    "@acala-network/api": "^0.7.4-20"
```
Comment (xlc) [inline packages/api-mobx/package.json:25]: why not latest version?

File: package.json (hunk 1 above)
Comment (ermalkaleci) [reply package.json:22]: It's not downgrade actually. Changed to the latest on a previous commit then to this one which matches with acala api

File: packages/api-mobx/package.json (hunk 2 above)
Comment (ermalkaleci) [reply packages/api-mobx/package.json:25]: This is used for testing. I think this is the latest one. Let me check

**Pull request merged**